Order page targeting properties to make it easier to compare to DCR

arelra opened the pull request:
## What does this change?

Order page targeting properties to make it easier to compare to DCR

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

zekehuntergreen commented on static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js:
```diff
@@ -264,41 +264,41 @@ const rebuildPageTargeting = () => {
+            ms: formatTarget(page.source),
+            permutive: getPermutiveSegments(),
+            pv: config.get('ophan.pageViewId'),
+            // round video duration up to nearest 30 multiple
```
this comment applies to the `vl` key, right?

arelra replied:
Good 👀

ashishpuliyel replied:
Booo. You beat me to it!

ashishpuliyel reviewed (changes requested):
~~The comment             // round video duration up to nearest 30 multiple
is no longer near the actual param (vl). Other than that it looks fine :)~~
Zeke already spotted this above

arelra commented:
> ~The comment // round video duration up to nearest 30 multiple is no longer near the actual param (vl). Other than that it looks fine :)~
> Zeke already spotted this above

Fixed and updated the other comments to be a bit more descriptive
